spec/person_spec.rb

```diff
@@ -0,0 +1,26 @@
+require_relative '../modules/person'
+
+RSpec.describe Person do
+  let(:person) { Person.new(18, '1234-5678', 'John Smith', parent_permission: true) }
+  let(:book) { double('Book') }
+
+  describe '#can_use_services?' do
+    context 'when the person is of age' do
+      it 'returns true' do
+        expect(person.can_use_services?).to be true
+      end
+    end
+  end
+
+  describe '#to_json' do
+    it 'returns a JSON representation of the person' do
+      expect(person.to_json).to eq({
+        JSON.create_id => 'Person',
+        'id' => '1234-5678',
```

Comment:
- [ ] Great job so far 👏🏻 👏🏻 However, according to the project requirements, there should be unit tests for all the methods in all the classes, Kindly make sure the `of_age?, correct_name and add_rental` class person method has (at least) an 'it' block 👇🏻 check your person.rb file

```ruby
 def of_age?
    @age > 18
  end

  def correct_name
    @name
  end

  def add_rental(book, date = Date.today)
    @rentals.push(Rental.new(date, self, book))
  end
```